package-lock.json

```diff
@@ -1,9107 +1,8 @@
 {
   "name": "react-jsonschema-form",
   "version": "2.0.0-alpha.1",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
Consider reverting this file... or switching to node 16? I'm pretty sure the revert is perfectly fine.

Reply:
I think because all our lockfiles currently have `"lockfileVersion": 1,` set, I'd like to keep this one using `"lockfileVersion": 1,` for consistency. We should upgrade all pacakges to node 16 maybe in another PR, though I've faced issues when trying so before, so it might be a more involved process to do so.